### PR TITLE
ctl: the duration of the pause scheduler must be specified

### LINKED ISSUE #4420: Incorrect behavior when using `pd-ctl` to pause the scheduler
### What did you do?

`pd-ctl scheduler pause balance-hot-region-scheduler`

### What did you expect to see?

An error is reported, or the scheduler is paused permanently, or the schedule is paused for thirty minutes by default.

### What did you see instead?

Actually equivalent to executing `pd-ctl scheduler resume balance-hot-region-scheduler`.

### What version of PD are you using (`pd-server -V`)?

v5.3.0

### PR DESCRIPTION
**Pull request by HunDunDM**
### What problem does this PR solve?

close #4420

### What is changed and how it works?

Separate the logic of `pause` and `resume`.

### Check List

Tests

- Unit test

Code changes

Side effects

Related changes

### Release note

```release-note
None
```

### CODE REVIEW

**Comment by rleungx** [comment]
How about adding a unit test?

File: tools/pd-ctl/pdctl/command/scheduler.go [hunk 1]
```diff
@@ -56,27 +56,43 @@ func NewPauseSchedulerCommand() *cobra.Command {
 	c := &cobra.Command{
 		Use:   "pause <scheduler> <delay>",
```
Comment (JmPotato) [inline tools/pd-ctl/pdctl/command/scheduler.go:57]: Should we add help info about the unit of the delay time?

Comment (HunDunDM) [reply tools/pd-ctl/pdctl/command/scheduler.go]: rename to `delay_second`

**Comment by HunDunDM** [comment]
> How about adding a unit test?

There have been separate tests for these two commands. Two error tests have been added.

**Review by lhy1024 — APPROVED** [review]

**Comment by bufferflies** [comment]
Should it need to modify http api also?

File: tools/pd-ctl/pdctl/command/scheduler.go [hunk 2]
```diff
@@ -54,29 +54,45 @@ func NewSchedulerCommand() *cobra.Command {
 // NewPauseSchedulerCommand returns a command to pause a scheduler.
 func NewPauseSchedulerCommand() *cobra.Command {
 	c := &cobra.Command{
-		Use:   "pause <scheduler> <delay>",
+		Use:   "pause <scheduler> <delay_second>",
```
Comment (bufferflies) [inline tools/pd-ctl/pdctl/command/scheduler.go:57]:
```suggestion
		Use:   "pause <scheduler> <delay_second>",
		Use:   "pause <scheduler> <delay_seconds>",
```

File: tools/pd-ctl/pdctl/command/scheduler.go [hunk 3]
```diff
@@ -54,29 +54,45 @@ func NewSchedulerCommand() *cobra.Command {
...
 }
 
-func pauseOrResumeSchedulerCommandFunc(cmd *cobra.Command, args []string) {
-	if len(args) != 2 && len(args) != 1 {
+func pauseSchedulerCommandFunc(cmd *cobra.Command, args []string) {
+	if len(args) != 2 {
+		cmd.Usage()
+		return
+	}
+	delay, err := strconv.Atoi(args[1])
```
Comment (bufferflies) [inline tools/pd-ctl/pdctl/command/scheduler.go:69]: the delay is int but the api suport int64. 
`
if err := h.PauseOrResumeScheduler(name, int64(t)); err != nil {
		h.r.JSON(w, http.StatusInternalServerError, err.Error())
		return
	}
`

**Review by bufferflies — APPROVED** [review]

**Pull request merged**